drawSvg/raster.py

```
@@ -1,11 +1,21 @@
 
 import io
+import warnings
+from .missing import MissingModule
+
 
 try:
     import cairosvg
-except (ImportError, OSError):
-    import warnings
-    from .missing import MissingModule
+except OSError as e:
+    msg = (
+        'Failed to import CairoSVG. '
+        'drawSvg will be unable to output PNG or other raster image formats. '
+        'See https://github.com/cduck/drawSvg#prerequisites for more details.\n'
+        f'Original OSError: {e}'
+    )
+    cairosvg = MissingModule(msg)
+    warnings.warn(msg, RuntimeWarning)
+except ImportError:
     msg = 'CairoSVG will need to be installed to rasterize images: Install with `pip3 install cairosvg`'
```

Do you want to add a similar error message here?

---

Yes. For completeness and to avoid masking original errors, I'll include the original import error and should be something like:


```
  msg = (
        'CairoSVG will need to be installed to rasterize images: Install with `pip3 install cairosvg`\n'
        'Original ImportError: {}'.format(e)
    )
```